ci: add github workflow

dignifiedquire opened the pull request.

dignifiedquire committed:
ci: add github workflow

dignifiedquire committed:
chore: happy clippy

dignifiedquire committed:
chore: fmt

dignifiedquire committed:
chore: happy clippy

Frando committed:
fix import test sort call

Frando commented:
I think the remaining clippy fail spotted my broken sort call which is fixed in #11

Frando committed:
fix: try fix path issues on windows

CI showed that the import test fails on windows. copied a util function
from iroh.

Frando commented:
The fail on windows is real, the import function didn't cater for windows paths. #12 tries to fix, let's see if it passes.

Frando committed:
chore: fmt

Frando committed:
fix: canonicalize path in test

dignifiedquire committed:
Merge pull request #12 from n0-computer/Frando/fix-import-windows

fix: path issues on windows
